example/pythonlib/basic/1-simple/build.mill

```diff
@@ -1,27 +1,59 @@
 package build
 import mill._, pythonlib._
 
-// Docs for PythonLib
 object foo extends PythonModule {
-  object bar extends PythonModule {
-    def pythonDeps = Seq("pandas==2.2.3", "numpy==2.1.3")
+  def pythonDeps = Seq("pandas==2.2.3", "numpy==2.1.3")
```

Comment:
Let's get rid of pandas and numpy in these examples; they're really heavyweight packages and take a long time to download and install. We use HTML templating in our Java/Scala/Kotlin examples, so let's do so here as well. Jinja2 seems like a reasonable choice